src/styles/blocks/catalog.scss

```diff
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $card-during-anim;
```

Comment:
Color needs to be animated as well as transform prop. Better create variable only for duration.

Reply:
agree

Reply:
what about color animation?